doc/_scripts/software_maturity/software_maturity_scanner.py

```diff
@@ -240,16 +240,20 @@ def find_experimental_symbols(logs: List[str]) -> Dict[str, Set[str]]:
         content = path.read_text()
         matches = re.findall(r"Experimental symbol ([A-Z0-9_]+) is enabled", content)
         if matches:
-            config_path = path.parent / "zephyr" / ".config"
+            config_path = path.parent / path.parts[-3] / "zephyr" / ".config"
```

Comment:
`path.parts[-3]` is not precisely clear, can it please be explained in a comment?

Reply:
comment added